### ISSUE TITLE
Training HyperParameters for Epipolar Transformer

### ISSUE THREAD

**Issue by HowieMa** [yihui-he/epipolar-transformers#15]
Hi, thank you for sharing such an awesome project. I am a little confused about your training hyperparameters. 
Based on your README, we should use [keypoint_h36m_zresidual_fixed.yaml](https://github.com/yihui-he/epipolar-transformers/blob/master/configs/epipolar/keypoint_h36m_zresidual_fixed.yaml#L24) to train the epipolar transformer to achieve 33.1 MPJPE. In that file, the max epoch is just **4**. However, in your paper, it is said that "The networks are trained for **20** epochs with batch size 16 and Adam optimizer [18]. Learning rate decays were set at 10 and 15 epochs.". Thus, which one is correct?

BTW, do I need to train the simple baseline on H36M firstly by:
`python main.py --cfg configs/benchmark/keypoint_h36m.yaml`
Then I can train the epipolar transformer with only 4 epochs by:  
`python main.py --cfg configs/epipolar/keypoint_h36m_zresidual_fixed.yaml`. 
Or I could directly train it from scratch and achieve your performance by: 
`python main.py --cfg configs/epipolar/keypoint_h36m_zresidual_fixed.yaml. `

Look forward to your reply, thanks!

**Comment by yihui-he** [comment]
both are correct. For 4 epochs schedule, [DATASETS.H36M.TRAIN_SAMPLE](https://github.com/yihui-he/epipolar-transformers/blob/6e155c0c006658d23d54071cf71523c28798698d/core/config.py#L190) is set to 0, resulting in a dataset 5 times larger.

it can be trained from scratch
